helpers/types.rs

```diff
@@ -55,3 +55,27 @@ macro_rules! construct_from {
 }
 
 construct_from!(u8, u16, u32, u64, usize, i8, i16, i32, i64);
+
+// Fee size that is charged for trading.
+// Denominated in parts per 1000.
+// ie 1 = 0.1%, 10 = 1%, 100 = 10%
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Encode, Decode)]
+#[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
+#[repr(u8)]
+pub enum FeeSize {
+    Normal = 3,
+    Stable = 1,
+}
+
+impl Default for FeeSize {
+    fn default() -> Self {
+        FeeSize::Normal
+    }
+}
+
+#[cfg(feature = "std")]
+impl StorageLayout for FeeSize {
```

Comment:
What does this do and why do we need that?

Reply:
Ask compiler 🙃

Reply:
Well, but this goes into a pair storage and stays there forever. Would be good to know wtf

Reply:
But this is used only during metadata run I think (`#[cfg(feature = "std")]). Even more confusing, but probably nothing dangerous